src/main/java/igrad/ui/AvatarSelectionPanel.java

```diff
@@ -74,17 +79,23 @@ private void initAvatarImgList() {
     private void showAvatarImgList() {
         // TODO: (Wayne) you could also display the Avatar names too, I had a hard time figuring that out
         for (AvatarImage avatarImg : avatarImgList) {
+            StackPane pane = new StackPane();
+            Label avatarName = new Label(avatarImg.getAvatarName());
             ImageView avatarDisplay = new ImageView();
 
             // TODO: (Wayne) try to find a way to set these preferences in the FXML files instead of here
-            avatarDisplay.setFitHeight(0.1 * primaryScreenBounds.getHeight());
-            avatarDisplay.setFitWidth(0.1 * primaryScreenBounds.getHeight());
-
+            avatarDisplay.setFitHeight(0.12 * primaryScreenBounds.getHeight());
+            avatarDisplay.setFitWidth(0.12 * primaryScreenBounds.getHeight());
             avatarDisplay.setImage(avatarImg);
-            GridPane.setRowIndex(avatarDisplay, avatarImg.getRowIndex());
-            GridPane.setColumnIndex(avatarDisplay, avatarImg.getColIndex());
-            avatarGridPane.getChildren().add(avatarDisplay);
+
+            avatarName.setStyle("-fx-font-family: 'Calibri Light'; -fx-font-size: 30; -fx-text-fill: white");
+            avatarName.setPadding(new Insets(200, 0, 0, 0));
```

Comment:
@dargohzy I'm not sure probably the padding set by this line was too much, and there's not enough space to display all the uwus (as it appeared on my screen). Reducing this helped, to fit all the uwus, but the animal names were all too big, please fix.